app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -79,34 +102,19 @@ export default function SearchResult({
 }: SearchResultProps) {
   const classes = useStyles();
   return (
-
-    <Card id={id} 
-         className={className} elevation={highlight ? 4 : undefined}>
+    <Card id={id} className={className} elevation={highlight ? 4 : undefined}>
       <CardContent>
         <UserSummary user={user} titleIsLink nameOnly>
           <div className={classes.statusLabelWrapper}>
             <div>
-              <Hidden smDown>
-                <CouchIcon className={classes.statusIcon} />
-              </Hidden>
-              <Typography
-                className={classes.statusLabel}
-                display="inline"
-                variant="body1"
-                color="primary"
-              >
+              <CouchIcon className={classes.statusIcon} />
+              <Typography display="inline" variant="body1" color="primary">
                 {hostingStatusLabels[user.hostingStatus]}
               </Typography>
             </div>
             <div>
-              <Hidden smDown>
-                <LocationIcon className={classes.statusIcon} />
-              </Hidden>
-              <Typography
-                className={classes.statusLabel}
-                display="inline"
-                variant="body1"
-              >
+              <LocationIcon className={classes.statusIcon} />
+              <Typography display="inline" variant="body1">
```

Comment:
Thanks for fixing the wrapping, but I think we should still hide the icons for xsDown. We want to support 320px width displays and with the icons at that width, you can't see much label. (That might mean this needs to be flex rather than grid.)

Reply:
This one is really tricky.
If you get rid of the icons, the labels lose their context, and in some cases is hard to guess what are they referring to
![Image 1](https://i.ibb.co/gzWyGFC/Captura-de-pantalla-de-2021-06-08-08-24-18.png)
Making it flex is also problematic, as when both texts don't fit, they push all the element down, breaking the layout
![Image 2](https://i.ibb.co/260HRJf/Captura-de-pantalla-de-2021-06-08-08-28-09.png)
This could be very problematic with languages that are not as economic as English when it comes to hosting.
![Image 2](https://i.ibb.co/k42zF3X/Captura-de-pantalla-de-2021-06-08-08-29-06.png)

Reply:
Good idea. Let's leave this for now then and ask design for some icons

Reply:
Fine, let me know if there's something pending here